modules/oe_theme_content_news/src/Plugin/PageHeaderMetadata/NewsContentType.php

```diff
@@ -98,9 +100,16 @@ public function getMetadata(): array {
       ];
     }
 
+    $news_type = $node->get('oe_news_types')->entity;
+    if ($news_type instanceof ConceptInterface) {
+      $cacheability = new CacheableMetadata();
+      $cacheability->addCacheableDependency($news_type);
+      $news_type = \Drupal::service('entity.repository')->getTranslationFromContext($news_type);
+      $news_type_label = $news_type->label();
+    }
     $timestamp = $node->get('oe_publication_date')->date->getTimestamp();
     $metadata['metas'] = [
-      $this->t('News'),
```

Comment:
Is this not required anymore? Let's maybe keep it as a fallback in case label is empty, which it will be the case for already existing content.